src/toxcore/dht/server/mod.rs

```diff
@@ -1114,6 +1118,14 @@ impl Server {
         });
         self.send_to(packet.ip_port.to_saddr(), next_packet)
     }
+    /// handle BootstrapInfo, respond with BootstrapInfo with version = 07032018
+    fn handle_bootstrap_info(&self, _packet: BootstrapInfo, addr: SocketAddr) -> IoFuture<()> {
+        let packet = DhtPacket::BootstrapInfo(BootstrapInfo {
+            version: 07032018,
```

Comment:
I think it's better to move version to constant, so we can easily find and update it.

Reply:
Ah! I forgot it